dev_docker/node_entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+npm install
+npm run setup
+npm run build-client
+npm run build
```

Comment:
only one build command is needed. should be able to remove `npm run build-client`